src/Microsoft.Sbom.Targets/GenerateSbomTask.cs

```diff
@@ -172,6 +163,22 @@ public override bool Execute()
         }
     }
 
+    /// <summary>
+    /// Sanitize arguments that are considered the "coordinates" of
+    /// an SBOM, such as package name, version, and supplier information.
+    /// </summary>
+    private void SanitizeBaselineArgs()
+    {
+        this.PackageSupplier = Remove_Spaces_Tabs_Newlines(this.PackageSupplier);
```

Comment:
I think we can add these lines to `ValidateRequiredParams`, in a similar way to what you did in `ValidateAndSanitizeNamespaceUriUniquePart`.

Reply:
Updated!